### PR TITLE
IDEMPIERE-3980

### PR DESCRIPTION
**Pull request by CarlosRuiz-globalqss**
https://idempiere.atlassian.net/browse/IDEMPIERE-3980

### CODE REVIEW

**Comment by hengsin** [comment]
on firefox, "Right Click > This Frame > Open Frame in New Tab" will execute the embedded script.

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/WImageDialog.java [hunk 1]
```diff
@@ -328,14 +350,25 @@ private void processUploadMedia(Media imageFile) {
 			return;
 
 		String fileName = imageFile.getName();
-		
+		String mimeType = MimeType.getMimeType(fileName);
+		if (! autoPreviewList.contains(mimeType))
+			throw new AdempiereException("File not allowed for uploading, just image types jpg/png/gif");
```
Comment (hengsin) [inline org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/WImageDialog.java:355]: should use ad_message here.

**Review by hengsin — CHANGES_REQUESTED** [review]

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/WImageDialog.java (hunk 1 above)
Comment (CarlosRuiz-globalqss) [reply org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/WImageDialog.java]: ready - committed message

**Review by hengsin — APPROVED** [review]
looks good

**Pull request merged**